framework-plugins/lisk-framework-forger-plugin/src/forger_plugin.ts

```diff
@@ -114,27 +122,57 @@ export class ForgerPlugin extends BasePlugin {
 		const options = objects.mergeDeep({}, config.defaultConfig.default, this.options) as Options;
 		this._channel = channel;
 
+		// eslint-disable-next-line new-cap
+		const { locale } = Intl.DateTimeFormat().resolvedOptions();
+
+		this._webhooks = new Webhooks(
+			{
+				'User-Agent': `lisk-framework-forger-plugin/0.1.0 (${os.platform()} ${os.release()}; ${os.arch()} ${locale}.${
+					process.env.LC_CTYPE ?? ''
+				}) lisk-framework/${options.version}`,
+			},
+			options.webhook,
+		);
 		this._forgerPluginDB = await getDBInstance(options.dataPath);
 
 		// eslint-disable-next-line @typescript-eslint/no-misused-promises
 		this._channel.once('app:ready', async () => {
 			this._app = initApi(options, this._channel, this.codec);
 
+			// eslint-disable-next-line no-void
+			void this._webhooks.handleEvent({
```

Comment:
maybe you could remove `void`?

Reply:
It's not possible as we discussed with @shuse2 not to wait of the resolution to avoid blocking the rest of the plugin working. In the future if this hooks are moved to its own plugin probably we should in that context wait for it.